addons/sale_project/models/project.py

```diff
@@ -378,6 +378,9 @@ def _get_profitability_sale_order_items_domain(self, domain=None):
             domain,
         ])
 
+    def _get_already_included_profitability_invoice_line_ids(self):
+        return super()._get_already_included_profitability_invoice_line_ids()
+
```

Comment:
Useless, since we just call the parent method

Reply:
mb :see_no_evil:  forgot to remove it